code/modules/mob/living/living.dm

```diff
@@ -810,6 +810,35 @@
 
 	return stat != DEAD
 
+//proc used to heal a mob, but only damage types specified.
+/mob/living/proc/specific_heal(blood_amt = 0, brute_amt = 0, fire_amt = 0, tox_amt = 0, oxy_amt = 0, clone_amt = 0, organ_amt = 0, stam_amt = 0, specific_revive = FALSE, specific_bones = FALSE)
+	if(iscarbon(src))
```

Comment:
as on discord, use proper OOP
![image](https://user-images.githubusercontent.com/75460809/213033018-d044b86e-bb12-436f-bf75-9c0d7c8717db.png)

Reply:
I think this is fixed?